pkg/generate/generate.go

```diff
@@ -0,0 +1,59 @@
+package generate
+
+import (
+	"context"
+	"fmt"
+	"io"
+)
+
+type (
+	Generator interface {
+		Evaluate(ctx context.Context) ([]string, error)
```

Comment:
A `Generator` with just an `Evaluate` method read weird to me. Would `Generate` fit better?

Reply:
Yeah, that reads a lot better.